### DISCUSSION TITLE
Realtime chart

### DISCUSSION THREAD

**Discussion by azrin1972** [apexcharts/apexcharts.js#4029] [Q&A]
How best to use apex chart with data updated every second? I’m currently using updateSeries but every time I update the data, I loose the zoom range and also the download burger menu will automatically close. Filtering data using the clicking on the label also resets right after I call updateSeries

**Comment by it-xtech-dev** [comment]
It seems your solution is recommended one: https://apexcharts.com/javascript-chart-demos/line-charts/realtime/
Maybe you should push this as an issue?
